src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -576,5 +576,44 @@ def add_noise(
         noisy_samples = original_samples + noise * sigma
         return noisy_samples
 
+    def get_velocity(
```

Comment:
can we add a `#Copied from` statement?

Reply:
see https://huggingface.co/docs/diffusers/v0.27.2/en/conceptual/contribution#copied-from-mechanism

Reply:
Added!

Reply:
can you run `make fix-copies`?

Reply:
> can you run `make fix-copies`?

Can you help me run it? I don't have the installation available on the cluster.

Reply:
@yiyixuxu I took a look again. For EulerDiscrete the timesteps are not supposed long. So, we cannot use DDPM `get_velocity()` here I think.

Reply:
> @yiyixuxu I took a look again. For EulerDiscrete the timesteps are not supposed long. So, we cannot use DDPM `get_velocity()` here I think.

Sorry, my implementation is actually not copied from DDPM. In fact, it is not copied from anywhere. The implementation as is takes care of the continuous nature of the timestep.

Reply:
oh thanks!
yeah it looks good to me1